dali/operators/random/philox.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_RANDOM_PHILOX_H_
+#define DALI_OPERATORS_RANDOM_PHILOX_H_
+
+#include <cassert>
+#include <cstdint>
+#include "dali/core/api_helper.h"
+
+namespace dali {
+
+class Philox4x32_10 {
+ public:
+  void skipahead(uint64_t n) {
+    if (advance(n))
+      recalc_output();
+  }
+
+  void skipahead_sequence(uint64_t n) {
+    if (advance_sequence(n))
+      recalc_output();
+  }
```

Comment:
`skipahead` and `skipahead_sequence` are not tested

Reply:
Added.